### PR TITLE
builtin: add utc-time built-in function

### PR DESCRIPTION
**Pull request by keviswang**

### CODE REVIEW

File: plan/typeinferer_test.go [hunk 1]
```diff
@@ -149,6 +149,7 @@ func (ts *testTypeInferrerSuite) TestInferType(c *C) {
 		{"curtime(2)", mysql.TypeDuration, charset.CharsetBin},
 		{"current_timestamp()", mysql.TypeDatetime, charset.CharsetBin},
 		{"utc_timestamp()", mysql.TypeDatetime, charset.CharsetBin},
+		{"utc_time()", mysql.TypeDatetime, charset.CharsetBin},
```
Comment (keviswang) [inline plan/typeinferer_test.go]: fix this. It is TypeDuration.

**Comment by shenli** [comment]
@keviswang CI failed.

**Comment by shenli** [comment]
@keviswang Any update?

**Comment by XuHuaiyu** [comment]
@keviswang 
hi, any updates?

**Comment by coocood** [comment]
@keviswang 
I'm sorry this function has been implemented already.

**Pull request closed without merging**